<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RigConfigurationDialog</class>
 <widget class="QDialog" name="RigConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>265</y>
     <width>160</width>
     <height>26</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidgetRigConnections">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>140</width>
     <height>38</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pageComPort">
    <widget class="QLabel" name="labelComPort">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>9</y>
       <width>44</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Com Port</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBoxComPort">
     <property name="geometry">
      <rect>
       <x>59</x>
       <y>9</y>
       <width>74</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pageUSB">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>USB Port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxUSBPort"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonTestRig">
   <property name="geometry">
    <rect>
     <x>314</x>
     <y>121</y>
     <width>77</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>402</width>
     <height>58</height>
    </rect>
   </property>
   <property name="title">
    <string>S-Meter</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QProgressBar" name="ProgrSigStrength">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="CheckBoxEnableSMeter">
      <property name="text">
       <string>Enable</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RigConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RigConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
